tests/strategies/geometry.py

```diff
@@ -47,10 +48,10 @@
 finite_floats = to_finite_floats()
 positive_floats = to_finite_floats(min_value=ABS_TOL,
                                    exclude_min=True)
-coordinates = tuples(finite_floats, finite_floats)
+coordinates = planar.points(finite_floats)
 angles = to_finite_floats(min_value=-360,
                           max_value=360)
-points = builds(Point, finite_floats, finite_floats)
+points = builds(Point, coordinates)
```

Comment:
will it work? building from `tuple`s is really supported?

Reply:
Yes. According to [Shapely docs on `Point`](https://shapely.readthedocs.io/en/latest/manual.html#points):

> The _Point_ constructor takes positional coordinate values or point tuple parameters.
> ```
> >>> from shapely.geometry import Point
> >>> point = Point(0.0, 0.0)
> >>> q = Point((0.0, 0.0))
> ```